src/Directory.Build.props

```diff
@@ -12,7 +12,7 @@
     <LangVersion>10</LangVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <AkkaVersion>1.5.14</AkkaVersion>
+    <AkkaVersion>1.5.22-beta1</AkkaVersion>
```

Comment:
Used a local beta that I created - will require Akka.NET v1.5.22 once that ships.